### ISSUE TITLE
Short text for logo on sign up is not appropriate

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4001]
## Area or Component
Sign up form

## Quick Summary
The SimpleReport logo has the short text "SimpleReport", but would more accurately be "SimpleReport logo".

### How to Replicate
Visit sign up form and inspect the logo at the top of the page

## Accessibility Violation
Short text alternative is not appropriate: https://cdc-devtools.dequecloud.com/rule-help/en/alt-text-short-text-not-meaningful

## Action Requested
Edit the short (alt) text for the logo

## Screenshots (optional)
![Screen Shot 2022-07-19 at 11.07.27 AM.png](https://images.zenhubusercontent.com/604699209bfd9306195da0d7/a2c6159d-9e73-46ac-b061-8396eaf1c41b)

## Original Deque Issue
  | alt-text-short-text-not-meaningful | The short text alternative for the complex image is not appropriate or meaningful. | Short text for the logo at the top is "SimpleReport" but would more accurately be "SimpleReport logo" | Short text alternative is not appropriate | https://cdc-devtools.dequecloud.com/rule-help/en/alt-text-short-text-not-meaningful | serious | Yes | No | Images | ["body > div#root:nth-of-type(1) > div.App:nth-of-type(1) > div#main-wrapper:nth-of-type(1) > div.card__background.bg-base-lightest:nth-of-type(2) > div.grid-container.maxw-mobile-lg.usa-section:nth-of-type(1) > div.card:nth-of-type(1) > header.display-flex.flex-column:nth-of-type(1) > img.flex-align-self-center.maxw-card-lg.width-full:nth-of-type(1)"] | The short text alternative for the complex image is not appropriate or meaningful. | <img class="flex-align-self-center maxw-card-lg width-full" src="/app/static/media/simplereport-logo-color.d759dad69912699b2a19efa057c9d3f2.svg" alt="SimpleReport"> | wcag2a,wcag111c | rnv7@cdc.gov | Sign up selector | https://www.simplereport.gov/app/sign-up/ |   | 2022-07-15T19:16:14.622Z |   |   |   |   |   |   |   |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
